lib/xboxrt/libc_extensions/string_ext_.h

```diff
@@ -9,6 +9,27 @@ static char *_strdup (const char *s)
     return strdup(s);
 }
 
+// Compared to their Win32 counterparts, these functions currently don't invoke
+// an invalid parameter handler routine as described in the MSDN, and instead
+// behave as if the invalid parameter handler allowed to continue execution.
+int _strnicmp (const char *s1, const char *s2, size_t n);
+int _stricmp (const char *s1, const char *s2);
+
+static int strnicmp (const char *s1, const char *s2, size_t n)
```

Comment:
These should be deprecated; [see MSDN](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/strnicmp-wcsnicmp?view=vs-2019)

> The Microsoft-specific function names strnicmp and wcsnicmp are deprecated aliases for the _strnicmp and _wcsnicmp functions. By default, they generate Compiler warning (level 3) C4996.

Then [see the description for C4996](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4996?view=vs-2019):

> Your code uses a function, class member, variable, or typedef that's marked deprecated. Symbols are deprecated by using a __declspec(deprecated) modifier, or the C++14 [[deprecated]] attribute.

Reply:
Done.